Bizyhood/ApiCalls.php

```diff
@@ -0,0 +1,359 @@
+<?php
+
+// Make sure we don't expose any info if called directly
+if ( !function_exists( 'add_action' ) ) {
+	echo 'Hi there!  I\'m just a plugin, not much I can do when called directly.';
+	exit;
+}
+
+/**
+ * This class contains all calls methods to be made to the Bizyhood API.
+ */
+  class Bizyhood_ApiCalls
```

Comment:
I think this class can be called just API and the file should be changed to that as well.